### PR TITLE
Removed unnecessary code

### PR DESCRIPTION
**Pull request by darialevko**
I removed unnecessary code. Please review.
Thanks.

### CODE REVIEW

File: js-tasks/5.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+		output+=y;
+	} else if (input[i] === "i") {
+		output+=z;
+	} else if (input[i] === "o") {
+		output+=c;
+	} else {
+		output+=input[i]
+	}
+}
+console.log(output);
```
Comment (ArcheGalla) [inline js-tasks/5.js:21]: why do we need, console log here?

Comment (darialevko) [reply js-tasks/5.js:21]: I don't know.

Comment (ArcheGalla) [reply js-tasks/5.js:21]: approved

**Review by ArcheGalla — APPROVED** [review]
Good job

**Pull request merged**